src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDate date;
+
+    public Deadline(String description, LocalDate date) {
+        super(description);
+        this.date = date;
+    }
+
+    public Deadline(String description, LocalDate date, boolean isDone) {
+        super(description, isDone);
+        this.date = date;
+    }
+    @Override
+    public String toString() {
+        return "[D] " + super.toString() + " (by: " + date.format(DateTimeFormatter.ofPattern("MMM d yyyy")) +")";
```

Comment:
Shouldn't there be a whitespace after the last "+" operator? 🤔
I have noticed this in your Event class too.